chaoscenter/graphql/server/server.go

```diff
@@ -148,8 +163,28 @@ func main() {
 	projectEventChannel := make(chan string)
 	go projects.ProjectEvents(projectEventChannel, mongodb.MgoClient, mongodbOperator)
 
-	log.Infof("chaos manager running at http://localhost:%s", utils.Config.HttpPort)
-	log.Fatal(http.ListenAndServe(":"+utils.Config.HttpPort, router))
+	log.Infof("graphql server running at http://localhost:%s", utils.Config.HttpPort)
+	go func() {
+		err := http.ListenAndServe(":"+utils.Config.HttpPort, router)
+		if err != nil {
+			logrus.Fatal(err)
+		}
+	}()
+	if enableHTTPSConnection {
+		// configuration of the certificate what we want
```

Comment:
do we need log here as well same as above

`log.Infof("graphql server running at https://localhost:%s", utils.Config.HttpsPort)`